prince-of-versions/build.gradle

```diff
@@ -32,13 +32,15 @@ android {
 dependencies {
     implementation fileTree(dir: 'libs', include: ['*.jar'])
     implementation 'com.github.zafarkhaja:java-semver:0.9.0'
-    implementation 'com.android.support:appcompat-v7:27.1.1'
+    compileOnly 'com.android.support:appcompat-v7:27.1.1'
```

Comment:
You probably want to share library versions between multiple modules. I suggest you expose them via ext so that you can use them in all modules.